package.json

```diff
@@ -46,6 +47,7 @@
     "gulp-uglify": "~3.0",
     "gulp-vinyl-zip": "~2.2",
     "handlebars": "~4.7",
+    "highlight.js": "9.18.3",
```

Comment:
❓ only needed because of a strange "Opal" error?

Reply:
No we can remove it nice catch !
I will do it and merge this PR !